src/setup_payload/SetupPayload.h

```diff
@@ -84,6 +84,9 @@ const int kTotalPayloadDataSizeInBits =
 
 const int kTotalPayloadDataSizeInBytes = kTotalPayloadDataSizeInBits / 8;
 
+// From 5.1.2.2. QR Code Version 3 (29x29), ECC=L
```

Comment:
This section number may become stale.

Also, the maximum size is not stated in the spec:

![image](https://user-images.githubusercontent.com/1313847/126423157-a9917c88-389f-4139-bc39-8a609ed3f0c7.png)

The Version 3 ECC=L example is just an example... Nevertheless, it is possible to compute the maximum size of the onboarding payload without TLV, but I would argue the best thing to do is "let the caller pass in the buffer size they are willing to use" and do not have an opinion here about the max. That way, the caller can make sure to pass a buffer large enough for the encoding they need. An onboarding payload for ECM may have the PBKDF salt/iterations included, and still be generatable, even with TLV, if size fits in buffer.